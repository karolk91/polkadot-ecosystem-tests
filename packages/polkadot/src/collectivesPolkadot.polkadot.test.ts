import { collectivesPolkadot, polkadot } from '@e2e-test/networks/chains'

import { collectivesChainE2ETests } from '@e2e-test/shared'

collectivesChainE2ETests(polkadot, collectivesPolkadot, { testSuiteName: 'collectives & polkadot' })
