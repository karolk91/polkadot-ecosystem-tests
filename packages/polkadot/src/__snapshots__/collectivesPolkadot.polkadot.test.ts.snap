// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`collectives & polkadot > whitelisting a call by fellowship > destination chain events 1`] = `
[
  {
    "data": {
      "callHash": "0x0101010101010101010101010101010101010101010101010101010101010101",
    },
    "method": "CallWhitelisted",
    "section": "whitelist",
  },
  {
    "data": {
      "id": "(redacted)",
      "origin": {
        "Ump": {
          "Para": "(rounded 1000)",
        },
      },
      "success": true,
      "weightUsed": {
        "proofSize": 10000,
        "refTime": "(rounded 1000000000)",
      },
    },
    "method": "Processed",
    "section": "messageQueue",
  },
]
`;

exports[`collectives & polkadot > whitelisting a call by fellowship > source chain events 1`] = `
[
  {
    "data": {
      "fees": [
        {
          "fun": {
            "Fungible": "(rounded 300000000)",
          },
          "id": {
            "interior": "Here",
            "parents": 1,
          },
        },
      ],
      "paying": {
        "interior": {
          "X1": [
            {
              "Plurality": {
                "id": "Technical",
                "part": "Voice",
              },
            },
          ],
        },
        "parents": 0,
      },
    },
    "method": "FeesPaid",
    "section": "polkadotXcm",
  },
  {
    "data": {
      "destination": {
        "interior": "Here",
        "parents": 1,
      },
      "message": [
        {
          "UnpaidExecution": {
            "checkOrigin": null,
            "weightLimit": "Unlimited",
          },
        },
        {
          "Transact": {
            "call": {
              "encoded": "0x17000101010101010101010101010101010101010101010101010101010101010101",
            },
            "originKind": "Xcm",
            "requireWeightAtMost": {
              "proofSize": 10000,
              "refTime": 1000000000,
            },
          },
        },
      ],
      "messageId": "(redacted)",
      "origin": {
        "interior": {
          "X1": [
            {
              "Plurality": {
                "id": "Technical",
                "part": "Voice",
              },
            },
          ],
        },
        "parents": 0,
      },
    },
    "method": "Sent",
    "section": "polkadotXcm",
  },
]
`;
